modules/ROOT/pages/custom-response-policy-example.adoc

```diff
@@ -150,7 +174,10 @@ In the `<dependencies>` section of the policy's pom, include the following depen
 </dependency>
 ----
 
-Next, in the `template.xml` file, we can start including the following code:
+=== Add Conditions in Template.XML File
+
+*<Lorena, why do they need to do this? How does it help?>*
```

Comment:
The idea is to explain a little bit the skeleton of a custom policy, and how does it work.

Reply:
I believe that the importance of this section is to help them understand the structure of the policy template, and make a connection between the YAML file and Handlebars.

That what they define in the YAML file, will be used to complete some fields in the XML (for example, the expressions that will be used), or to add an entire block of code, etc.

Reply:
Done! Thanks!